wwwroot/src/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "A messy wiki that's only useful for wetfish",
   "author": "Rachel Fish <rachel@wetfish.net> (https://wetfish.net)",
   "dependencies": {
+    "@ruffle-rs/ruffle": "^0.1.0-nightly.2024.1.2",
```

Comment:
Can we use a stable version of ruffle instead of a random nightly build?

Reply:
Nevermind, apparently ruffle.js doesn't have an official release yet :sweat_smile:

Reply:
Discussion from ruffle.js discord

![image](https://github.com/wetfish/wiki/assets/1670549/a2e17658-7212-44e4-98a5-c90c4c5fb817)